### PR TITLE
Revert "ci: fix regression test workflow"

### PR DESCRIPTION
**Pull request by fukinbubba**
Reverts 0xPolygonHermez/zkevm-bridge-service#704

### CODE REVIEW

**Comment by fukinbubba** [comment]
.github\workflows\regression-tests.yml

**Review by FKNCOIN — COMMENTED** [review]
[]()ok

File: .github/workflows/regression-tests.yml [hunk 1]
```diff
@@ -2,82 +2,34 @@ name: Regression Tests
 
 on:
   pull_request:
```
Comment (FKNCOIN) [inline .github/workflows/regression-tests.yml:4]: Pull

**Review by FKNCOIN — COMMENTED** [review]
Ok

**Pull request closed without merging**